### DISCUSSION TITLE
Fields no longer required

### DISCUSSION THREAD

**Discussion by Ginger228** [CMSgov/price-transparency-guide#326] [Q&A] [answered]
@shaselton-usds 
Per the most recent schema the following fields show as not required- is this correct? 

- **version** | Version | String | The version of the schema for the produced information | No
- **description** | Description | String | Brief description of the item/service | No
- **Bundled_code**s | Bundled Codes | Array | This is an array of bundle code objects. This array contains all the different codes in a bundle if bundle is selected for negotiation_arrangement | No
- **covered_services |** Covered Service | Array | This is an array of covered services objects. This array contains all the different codes in a capitation arrangement if capitation is selected for negotiation_arrangement | No

Thanks.

**Comment by shaselton-usds** [comment, marked as answer]
@Ginger228 Closing out old discussions -- there was a documentation bug that was fixed post v1.0 for the `description` attribute. That was fixed from "no" to "yes". Nothing actually changed in the schema post v1.0 on that front.
